docs/getting_started.rst

```diff
@@ -0,0 +1,296 @@
+
+    # Subset to a specific set of variables
+    ds[['variant_allele', 'call_genotype']]
+
+    # Extract a single variable
+    ds.call_genotype[:3, :3]
+
+    # Access the array underlying a single variable (this would return dask.array.Array if chunked)
+    ds.call_genotype.data[:3, :3]
+
+    # Access the alleles corresponding to the calls for the first variant and sample
+    allele_indexes = ds.call_genotype[0, 0]
+    allele_indexes
+
+    ds.variant_allele[0, allele_indexes]
+
+    # Get a single item from an array as a Python scalar
+    ds.sample_id.item(0)
+
+Larger subsets of data can be visualized and/or summarized through various
+sgkit utilities as well as the Pandas/Xarray integration:
+
+.. ipython:: python
+
+    import sgkit as sg
+
+    # Show genotype calls with domain-specific display logic
+    sg.display_genotypes(ds, max_variants=8, max_samples=8)
+
+    # A naive version of the above is also possible using only Xarray/Pandas and
+    # illustrates the flexibility that comes from being able to transition into
+    # and out of array/dataframe representations easily
+    (ds.call_genotype[:5, :5].to_series()
+        .unstack().where(lambda df: df >= 0, None).fillna('.')
+        .astype(str).apply('/'.join, axis=1).unstack())
+
+    # Show call rate distribution for each variant using Pandas
+    df = ~ds.call_genotype_mask.to_dataframe()
+    df.head(5)
+
+    call_rates = df.groupby('variants').mean()
+    call_rates
+
+    @savefig call_rate_example.png width=6in height=3in
+    call_rates.plot(kind='hist', bins=24, title='Call Rate Distribution', figsize=(6, 3))
+
+This last example alludes to representations of missing data that are explained further in :ref:`missing_data`.
+
+Genetic methods
+~~~~~~~~~~~~~~~
+
+Genetic methods in sgkit are nearly always applied to individual ``Dataset`` objects.  For a full list of
+available methods, see :ref:`api_methods`.
+
+In this example, the ``variant_stats`` method is applied to a dataset to compute a number of statistics
+across samples for each individual variant:
+
+.. ipython:: python
+
+    sg.variant_stats(ds, merge=False)
+
+There are two ways that the results of every function are handled -- either they are merged with the provided
+dataset or they are returned in a separate dataset.  See :ref:`dataset_merge` for more details.
+
+
+.. _missing_data:
+
+Missing data
+~~~~~~~~~~~~
+
+Missing data in sgkit is represented using a sentinel value within data arrays
+(``-1`` in integer arrays and ``NaN`` in float arrays) as well as a companion boolean mask array
+(``True`` where data is missing). The sentinel values are often more useful when implementing compiled
+operations while the boolean mask array facilitates user operations in a higher level API like Xarray or Numpy.
+
+This example shows how either can be used, though users should prefer the mask array where possible since
+its on-disk representation is typically far smaller after compression is applied.
+
```

Comment:
This gets hard-core pretty quickly. I wonder if we should put in an example where missing data is handled automatically by the library first? Users might get the impression they need to know what numba and jitting are to use this.

Reply:
https://github.com/pystatgen/sgkit/pull/278/commits/ae349eea8e8b2094f7d55bf282e5c15d74c4a84b#diff-9bc8bf6e8d9db020cefce1fbc0426795R196

Reply:
FYI: https://github.com/pystatgen/sgkit/pull/278#issuecomment-700997663